<launch>
  <!-- Buzzer Service Server-->
  <node name="buzzer_server" pkg="buzzer" type="buzzer_server.py" output="screen"/>
  
  <!-- Camera Services Servers-->
  <node name="camera_server_switcher" pkg="camera" type="camera_switch.py" output="screen"/>
  <node name="camera_find_player_server" pkg="camera" type="find_player_server.py" output="screen"/>
  <node name="camera_finger_counter_server" pkg="camera" type="finger_counter_server.py" output="screen"/>
  <node name="color_selection_server" pkg="camera" type="color_selection_server.py" output="screen"/>
  <node name="wheel_server" pkg="camera" type="wheel_server.py" output="screen"/>
  
  <!-- IR Remote Talker-->
  <node name="ir_remote_talker" pkg="ir_remote" type="ir_remote_talker.py" output="screen"/>
  
  <!-- LEDs Service Server -->
  <node name="led_server" pkg="led" type="led_server.py" output="screen"/>
  
  <!-- Motors Action Server-->
  <node name="motor_action_server" pkg="motor" type="motor_action_server.py" output="screen"/>
  
  <!-- Servos Action Server -->
  <node name="servo_action_server" pkg="servo" type="servo_action_server.py" output="screen"/>
  
  <!-- PITCH DIALOG BOX -->
  <!-- Will be launched in another terminal window thanks to launch-prefix="gnome-terminal command" -->
  <node name="pitch_client" pkg="init" type="init_main.py" output="screen" launch-prefix="gnome-terminal --command"/>
</launch>
